lib/command/src/Obelisk/Command.hs

```diff
@@ -366,6 +368,7 @@ ob = \case
     DeployCommand_Update -> deployUpdate "."
     DeployCommand_Test (platform, extraArgs) -> deployMobile platform extraArgs
   ObCommand_Run -> run
+  ObCommand_Profile basePath -> profile basePath
```

Comment:
Would be cleaner to resolve the default for `basePath` here instead of inlining it later.

Reply:
Unfortunately I don't think we know the root path yet. We also need a timestamp which isn't possible in the applicative.

Reply:
I mean just describe it in words

Reply:
See how `DeployCommand_Init` is handled as an example of how to do this.

Reply:
Done in 9ec5d2b6